test/unittests/VisitTest.php

```diff
@@ -53,37 +53,150 @@ protected function setUp()
+                                       3,
+                                       1,
+                                      ),
+                                      array(
+                                       $v3,
+                                       3,
+                                       3,
+                                      ),
+                                      array(
+                                       $v4,
+                                       2,
+                                       3,
+                                      ),
+                                      array(
+                                       $v4,
+                                       2,
+                                       4,
+                                      ),
+                                      array(
+                                       $v4,
+                                       3,
+                                       3,
+                                      ),
+                                      array(
+                                       $v5,
+                                       2,
+                                       3,
+                                      ),
+                                      array(
+                                       $v5,
+                                       2,
+                                       4,
+                                      ),
+                                      array(
+                                       $v5,
+                                       3,
+                                       3,
+                                      ),
+                                      array(
+                                       $v6,
+                                       2,
+                                       3,
+                                      ),
+                                      array(
+                                       $v6,
+                                       2,
+                                       4,
+                                      ),
+                                      array(
+                                       $v6,
+                                       3,
+                                       3,
+                                      ),
                                      );
 
     }
 
     function testVisit()
     {
-       $visit_name = "Visit 1";
-       $visit = new Visit($visit_name);
-       $this->assertEquals($visit_name, $visit->getName(), "the name of the visit does not match");
+        $visit_name = "Visit 1";
+        $visit      = new Visit($visit_name);
+        $this->assertEquals($visit_name, $visit->getName(), "the name of the visit does not match");
     }
 
     function testAllVisit()
     {
         $visits = $this->_visitController->getAllVisits();
         $this->assertEquals($this->_listOfVisit, $visits, "the name of the visit does not match value in DB");
```

Comment:
@driusan this assertequals seems to be comparing arrays in an ordered fashion which forced me to add an `order by` clause to the visitController for the array returned by the controller to match the one here.

I'm not sure thats normal behaviour...

Same goes for the one remaining error on the PR for function `testVisitsProjects`  the ordering seems to be the issue